README.md

```diff
@@ -65,44 +65,44 @@ run the server from the live-server extinction
 
 to see this project visit this [link](https://danielf-canon.github.io/Portfolio/)
 
-## 👥 Authors <a name="authors"></a>
+## 👥 Authors `<a name="authors"></a>`
 
 Daniel Felipe Cañón
 
 - Twitter: [Twitter](https://twitter.com/canonsanchezd)
 - GitHub: [@githubhandle](https://github.com/danielf-canon)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/daniel-felipe-canon-sanchez)
 
-## 🔭 Future Features <a name="future-features"></a>
+## 🔭 Future Features `<a name="future-features"></a>`
 
 - *add som interactivity to the website*
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing <a name="contributing"></a>
+## 🤝 Contributing `<a name="contributing"></a>`
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## ⭐️ Show your support <a name="support"></a>
+## ⭐️ Show your support `<a name="support"></a>`
 
 thanks for visit my first portfolio.
 Thank you for supporting me in this
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+## 🙏 Acknowledgments `<a name="acknowledgements"></a>`
 
 I would like to thank every persone that encourge me to start this journey
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 📝 License <a name="license"></a>
+## 📝 License `<a name="license"></a>`
 
 MIT License
 
```

Comment:
Greeting!

Kindly, I suggest you remove the `License` test here, to make your README look more professional. 😄